demos/top-app-bar.html

```diff
@@ -232,6 +238,23 @@ <h4 class="mdc-typography--subheading2 mdc-typography--adjust-margin">Short Top
           }
         });
 
+        // Top App Bar Specific Options
+        prominentCheckbox.addEventListener('change', function()  {
+          appBarEl.classList[this.checked ? 'add' : 'remove']('mdc-top-app-bar--prominent');
+
+          shortCheckbox.disabled = this.checked;
+        });
+
+        fixedCheckbox.addEventListener('change', function()  {
+          appBarEl.classList[this.checked ? 'add' : 'remove']('mdc-top-app-bar--fixed');
```

Comment:
Nit: You might also want to toggle/remove `mdc-top-app-bar--fixed-scrolled` here, if it's not too hard to do.